setup.py

```diff
@@ -4,11 +4,8 @@
 import os
 import sys
 
-if os.environ.get("SENTRY_PYTHON3") == "1" and sys.version_info[:2] != (3, 6):
-    sys.exit("Error: Sentry [In EXPERIMENTAL python 3 mode] requires Python 3.6.")
-
-if os.environ.get("SENTRY_PYTHON3") != "1" and sys.version_info[:2] != (2, 7):
-    sys.exit("Error: Sentry requires Python 2.7.")
+if os.environ.get("SENTRY_PYTHON2") != "1" and sys.version_info[:2] != (3, 6):
```

Comment:
This change will render all nightly builds of Sentry self-hosted useless. Please hold on for the changes in `setup.py` or don't require an extra env var for Python 2 for now.

Reply:
Quite surprised that this did not break GCB builds.

Reply:
Oh wait never mind. This only applies at the packaging step, which is already using Python 3.

Reply:
Can I assume we're good here?

Reply:
In other words, can I dismiss the request for change?